src/components/Card/Card.tsx

```diff
@@ -13,38 +21,61 @@ export interface CardProps extends BoxProps {
    * visually subdue the appearance of the entire card.
    */
   subdued?: boolean;
+  /**
+   * If defined as a prop, all themeable styling will be removed.
+   * Radius of the Card's corners
+   */
+  radius?: BorderRadiusSize | ResponsiveProp<BorderRadiusSize>;
+  /**
+   * If defined as a prop, all themeable styling will be removed.
+   * The size of the drop shadow applied to the Card
+   */
+  shadow?: BoxShadowSize | ResponsiveProp<BoxShadowSize>;
 }
 
-const CardBaseComponent: React.FC<CardProps> = React.forwardRef((
-  {
-    children,
-    subdued,
-    className = undefined,
-    overflow = 'hidden',
-    display = 'block',
-    width = '100',
-    ...restProps
-  },
-  ref,
-) => {
-  const classes = classNames(styles.card, className,
+const CardBaseComponent: React.FC<CardProps> = React.forwardRef(
+  (
     {
-      [styles.subdued]: subdued,
-    });
+      background = undefined,
+      children,
+      subdued,
+      className = undefined,
+      overflow = 'hidden',
+      display = 'block',
+      radius = undefined,
+      shadow = undefined,
+      width = '100',
+      ...restProps
+    },
+    ref,
+  ) => {
+    const useTheme = background === undefined && radius === undefined && shadow === undefined;
```

Comment:
Background, radius, and shadow are themeable component design tokens, so if any of these are passed as props, don't apply the theme styles.